tests/integration/ambient/testdata/only-hbone.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
I am actually not sure this test makes sense...

Arguably, Istio's behavior here is broken (by changing the port netpol is enforced on). By adding this test, arguably we are forcing other implementations to be broken in the same way?

We don't have "conformance" yet, but there has been a lot of discussions around ambient conformance

Reply:
> Arguably, Istio's behavior here is broken (by changing the port netpol is enforced on). By adding this test, arguably we are forcing other implementations to be broken in the same way?

To your point, we don't have conformance yet, and these tests aren't necessarily conformance gates.

I don't really want to undertest the current impl because we are worried conformance decisions might require us to change test code - we can change test code. Tests can only ever encode expectations about the current state anyway and are not set in stone.